applications/power/power_service/power.c

```diff
@@ -126,6 +171,11 @@ static void power_check_charging_state(Power* power) {
 
 static bool power_update_info(Power* power) {
     PowerInfo info;
+	
```

Comment:
the problem is, we should not read int filesystem every 1000ms
thats a very bad implementation, i looked around to re-do it in better way, but doesn’t found a way,
if you can fix it, I will accept that, otherwise other parts are OK